### ISSUE TITLE
Microsoft Store page shows 2.x (but installs 1.x) on older Windows 10 machines

### ISSUE THREAD

**Issue by brutlern** [File-New-Project/EarTrumpet#237]
Opened Windows Store, found EarTrumpet, says it's version 2.0.2.0, installed it, and instead version 1.5.3.0 gets installed.

NVM: Just found out that I need latest Windows version. You should definitely change the store page to specify that version is 17134+ recommended for full features. Currently recommended version is 14342+ which is the same as the minimum version. So I guess the bug is "Store page not updated with proper requirements"

**Comment by riverar** [comment]
Sadly, this is a Microsoft Store limitation. We understand it's confusing, we're working on it!

**Comment by anton-rudeshko** [comment]
I've stumbled upon this too. How one get 17134+ Windows version? Is that some kind of beta / insiders preview?

**Comment by riverar** [comment]
Check for updates via Windows Update or you can trigger it manually via https://www.microsoft.com/en-us/software-download/windows10

**Comment by brutlern** [comment]
@anton-rudeshko It's the April 2018 Feature Update, also know as version 1803. It's a big update, most people should have already got it automatically, but in my case, I delayed it for 90 days because a lot of people reported problems with it. But I installed it (just ran "Check for Updates" and it showed up as "Feature update to Windows 10, version 1803") and had no problems, had to manually update EarTrumpet after that, now it's at 2.0.3.0.
